api/models/helpers.py

```diff
@@ -0,0 +1,29 @@
+import abc
+
+from django.db.backends.sqlite3.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+class DataMigrationMaster(abc.ABC):
```

Comment:
```suggestion
class DataMigrationMaker(abc.ABC):
```
We can make the name more predictable, consistent with `makemigrations`.

Reply:
Fixed